docs/src/main/sphinx/functions/aggregate.md

```diff
@@ -241,28 +241,33 @@ results in:
  200 | b,c
 ```
 
-This aggregation function can be also used with the `FILTER` keyword to specify
-which rows are processed during the `listagg` aggregation:
+This aggregation function supports 
```

Comment:
It is important to underline what `FILTER` does actually (we already have documentation for it - so no need to repeat ourselves here).

Doing the `GROUP BY` showcases that the `listagg` aggregation function still takes into account the tuples which don't match the `FILTER` condition.

Reply:
You need to explicitly say that in the docs, not just as comment in the PR